pkg/utils/utils.go

```diff
@@ -105,6 +105,19 @@ func LabelsForAerospikeClusterRack(clName string, rackID int) map[string]string
 	return labels
 }
 
+// MergeLabels merges operator an user defined labels
+func MergeLabels(operatorLabels, userLabels map[string]string) (map[string]string, error) {
+	for label, value := range userLabels {
+		_, ok := operatorLabels[label]
+		if ok {
+			return nil, fmt.Errorf(
```

Comment:
Add this validation check in validationWebhook. Request should not come here if this check fails there.

Reply:
Done